SiteAssets/pages/3.0/scripts/PROCESS.CONTROLLER.js

```diff
@@ -4575,11 +4575,11 @@ function hazardreviewbuttonaction() {
                                             }).done(r => {
                                                 let item = ((r.d.results[0].cdmRAMS != null));
                                                 if (item) {
-                                                    tdata.push("cdmCurrentStatus|Under Construction Manager review");
+                                                    tdata.push(`"cdmCurrentStatus|${configData["Workflow"][a]["nextWorkFlowState1"]}`); //Editable workflow config. Patrick Hsu, 30 Jan 2024
```

Comment:
The double quote mark after the special quote character needs to be removed

Reply:
This is present in a number of other places also

Reply:
Also lets just leave the workflow for RAMS hazard as it is

Reply:
I have checked and removed all the places with double quote marks after special quote